2020-09-20 13:08:53,637 - GlobalLog.py [line:123] - INFO: this is a logger info message
2020-09-20 13:08:53,638 - GlobalLog.py [line:124] - WARNING: this is a logger warning message
2020-09-20 13:08:53,638 - GlobalLog.py [line:125] - ERROR: this is a logger error message
2020-09-20 13:08:53,638 - GlobalLog.py [line:126] - CRITICAL: this is a logger critical message
2020-09-20 13:11:01 | ERROR | 这是一个Error问题
2020-09-20 13:11:01 | DEBUG | 这是一个Debug问题
2020-09-20 13:11:01 | INFO | 这是一个Info问题
2020-09-20 13:11:19 | ERROR | 这是一个Error问题
2020-09-20 13:11:19 | DEBUG | 这是一个Debug问题
2020-09-20 13:11:19 | INFO | 这是一个Info问题
2020-09-20 13:12:22,954 - Dir_Tools.py [line:78] - ERROR: [WinError 3] 系统找不到指定的路径。: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils'
2020-09-20 13:12:49,638 - Dir_Tools.py [line:78] - ERROR: [WinError 3] 系统找不到指定的路径。: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils'
2020-09-20 13:13:16,555 - Dir_Tools.py [line:78] - ERROR: [WinError 3] 系统找不到指定的路径。: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils'
2020-09-20 13:14:01,124 - Dir_Tools.py [line:103] - ERROR: [Errno 2] No such file or directory: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils\\copy\\aaaa.gzip'
2020-09-20 13:14:57,951 - Dir_Tools.py [line:103] - ERROR: [Errno 2] No such file or directory: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils\\copy\\aaaa.gzip'
2020-09-20 13:16:01,718 - Dir_Tools.py [line:88] - ERROR: [Errno 2] No such file or directory: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils\\copy'
2020-09-20 13:16:11,702 - Dir_Tools.py [line:88] - ERROR: [Errno 2] No such file or directory: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils'
2020-09-20 13:16:28,417 - Dir_Tools.py [line:88] - ERROR: [Errno 13] Permission denied: 'D:\\Work_Spaces\\PyCharm_Project\\UiAutomationFramework\\Utils'
2020-09-20 13:17:11,980 - Dir_Tools.py [line:88] - ERROR: [Errno 2] No such file or directory: 'D:\\Work_Spaces\\PyCharm_Project\\UiAutomationFramework\\Utils\\copy\\aaaa.zip'
2020-09-20 13:18:12,926 - Dir_Tools.py [line:60] - ERROR: [WinError 3] 系统找不到指定的路径。: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils'
2020-09-20 13:18:29,333 - Dir_Tools.py [line:60] - ERROR: [WinError 3] 系统找不到指定的路径。: 'E:\\WorkSpace\\PycharmProjects\\UiAutomationFramework\\Utils'
2020-09-20 13:31:31,595 - Dir_Tools.py [line:84] - ERROR: 'str' object has no attribute 'decode'
2020-09-20 13:32:43,411 - Dir_Tools.py [line:82] - ERROR: module 'os' has no attribute 'sep0'
2020-09-20 13:32:49,617 - Dir_Tools.py [line:82] - ERROR: name '_iterateExeZipFile' is not defined
2020-09-20 13:34:02,616 - Dir_Tools.py [line:82] - ERROR: name '_iterateExeZipFile' is not defined
2020-09-20 13:35:57,856 - Dir_Tools.py [line:81] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:36:23,980 - Dir_Tools.py [line:82] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:36:39,559 - Dir_Tools.py [line:81] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:37:38,027 - Dir_Tools.py [line:85] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:41:49,651 - Dir_Tools.py [line:88] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:42:10,998 - Dir_Tools.py [line:86] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:42:51,846 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:43:20,453 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:44:33,200 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:45:31,984 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:45:31,984 - Dir_Tools.py [line:90] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:45:38,648 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:45:51,281 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:45:51,281 - Dir_Tools.py [line:88] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:46:01,527 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:46:01,528 - Dir_Tools.py [line:88] - ERROR: module 'zipfile' has no attribute 'write'
2020-09-20 13:46:06,224 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:46:06,224 - Dir_Tools.py [line:88] - ERROR: 'str' object has no attribute 'compress_type'
2020-09-20 13:46:14,926 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:48:04,780 - Dir_Tools.py [line:67] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:48:42,799 - Dir_Tools.py [line:75] - ERROR: [Errno 2] No such file or directory: ''
2020-09-20 13:48:53,871 - Dir_Tools.py [line:75] - ERROR: [Errno 2] No such file or directory: ''
2020-09-20 13:49:26,099 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:49:40,082 - Dir_Tools.py [line:74] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:50:39,949 - Dir_Tools.py [line:71] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:51:01,447 - Dir_Tools.py [line:70] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:51:14,803 - Dir_Tools.py [line:66] - INFO: True
2020-09-20 13:51:14,803 - Dir_Tools.py [line:71] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:52:11,635 - Dir_Tools.py [line:66] - INFO: True
2020-09-20 13:52:11,636 - Dir_Tools.py [line:71] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:52:57,133 - Dir_Tools.py [line:63] - INFO: True
2020-09-20 13:52:57,133 - Dir_Tools.py [line:68] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:54:57,780 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:54:57,781 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:55:51,728 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:55:51,728 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:56:24,702 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:56:24,702 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:56:56,703 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:56:56,703 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:56:56,703 - Dir_Tools.py [line:73] - ERROR: list indices must be integers or slices, not str
2020-09-20 13:57:01,229 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:57:01,229 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:57:17,558 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:57:17,559 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:57:23,698 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:57:23,698 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:57:36,314 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:57:36,314 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:58:20,024 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:58:20,024 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:58:29,026 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:58:29,026 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:58:47,850 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:58:47,850 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 13:58:55,205 - Dir_Tools.py [line:59] - INFO: True
2020-09-20 13:58:55,205 - Dir_Tools.py [line:64] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 14:01:41,221 - Dir_Tools.py [line:60] - INFO: True
2020-09-20 14:01:41,222 - Dir_Tools.py [line:65] - INFO: 压缩存储后的路径：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\aaaa.gzip
2020-09-20 14:02:00,860 - Dir_Tools.py [line:156] - INFO: 请检查filepath是否正确！
2020-09-20 14:12:02,149 - Dir_Tools.py [line:174] - ERROR: 请检查filepath是否正确！
2020-09-20 14:18:02,973 - Dir_Tools.py [line:174] - ERROR: 请检查filepath是否正确！
2020-09-20 14:18:50,968 - Dir_Tools.py [line:178] - ERROR: aa
2020-09-20 14:19:37,854 - Dir_Tools.py [line:178] - ERROR: aa
2020-09-20 14:20:07,680 - Dir_Tools.py [line:178] - INFO: 开始创建文件夹！！！
2020-09-20 14:20:21,566 - Dir_Tools.py [line:178] - INFO: 文件夹不存在、自动创建中
2020-09-20 14:20:23,612 - Dir_Tools.py [line:178] - INFO: 文件夹不存在、自动创建
2020-09-20 14:21:05,134 - Dir_Tools.py [line:178] - INFO: 目录创建中：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA
2020-09-20 14:21:46,463 - Dir_Tools.py [line:178] - INFO: 目录D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA：创建中
2020-09-20 14:21:56,117 - Dir_Tools.py [line:178] - INFO: 目录创建中D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA：
2020-09-20 14:22:16,729 - Dir_Tools.py [line:178] - INFO: 目录D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA不存在，创建中
2020-09-20 14:22:27,980 - Dir_Tools.py [line:178] - INFO: 目录D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA不存在，开始创建
2020-09-20 14:28:31,160 - Dir_Tools.py [line:178] - INFO: 目录D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA不存在，开始创建！！！
2020-09-20 14:28:51,196 - Dir_Tools.py [line:185] - ERROR: D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA已存在，跳过创建！
2020-09-20 14:29:35,724 - Dir_Tools.py [line:185] - ERROR: D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA已存在，跳过创建！
2020-09-20 14:29:54,606 - Dir_Tools.py [line:181] - INFO: D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA，创建成功
2020-09-20 14:30:26,834 - Dir_Tools.py [line:181] - INFO: 目录：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA\aaa 创建成功！！！
2020-09-20 14:30:37,745 - Dir_Tools.py [line:181] - INFO: 目录：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Util\AA\aaa 创建成功！！！
2020-09-20 14:31:19,743 - Dir_Tools.py [line:185] - INFO: 目录：D:\Work_Spaces\PyCharm_Project\UiAutomationFramework\Utils\AA\aaa 创建成功！！！
2020-09-20 14:33:41,310 - GetSystemTime.py [line:146] - INFO: 1600583621.3105192
2020-09-20 14:33:41,318 - GetSystemTime.py [line:147] - INFO: 2020-09-20
2020-09-20 14:33:41,319 - GetSystemTime.py [line:148] - INFO: 14:33:41
2020-09-20 14:33:41,319 - GetSystemTime.py [line:149] - INFO: 2020-09-20 14:33:41
2020-09-20 14:33:41,319 - GetSystemTime.py [line:150] - INFO: 2020-09-20
2020-09-20 14:33:41,319 - GetSystemTime.py [line:151] - INFO: 14-33-41
2020-09-20 14:33:41,319 - GetSystemTime.py [line:152] - INFO: 2020-09-20 14:33:41
2020-09-20 14:33:41,319 - GetSystemTime.py [line:153] - INFO: 2020-09-30
2020-09-20 14:33:41,319 - GetSystemTime.py [line:154] - INFO: 2020-09-14
2020-09-20 14:33:41,319 - GetSystemTime.py [line:155] - INFO: 2020年9月17日
2020-09-20 14:33:41,319 - GetSystemTime.py [line:156] - INFO: 25时39分3秒
2020-09-20 14:33:41,319 - GetSystemTime.py [line:157] - INFO: 2020年9月17日25时39分3秒
2020-09-20 14:33:41,324 - GetSystemTime.py [line:131] - INFO: 中间共计31天
2020-09-20 14:33:41,324 - GetSystemTime.py [line:158] - INFO: ['2020-06-01', '2020-06-02', '2020-06-03', '2020-06-04', '2020-06-05', '2020-06-06', '2020-06-07', '2020-06-08', '2020-06-09', '2020-06-10', '2020-06-11', '2020-06-12', '2020-06-13', '2020-06-14', '2020-06-15', '2020-06-16', '2020-06-17', '2020-06-18', '2020-06-19', '2020-06-20', '2020-06-21', '2020-06-22', '2020-06-23', '2020-06-24', '2020-06-25', '2020-06-26', '2020-06-27', '2020-06-28', '2020-06-29', '2020-06-30', '2020-07-01']
2020-09-20 14:33:41,325 - GetSystemTime.py [line:159] - INFO: 1591008600
